README.md

```diff
@@ -23,11 +23,11 @@
 
 2.  A fontforge python script ([ligaturize.py](ligaturize.py)) that adds the ligatures from [Fira Code](https://github.com/tonsky/FiraCode) to a font without ligatures
 
-Here's a couple examples of the fonts generated: SF Mono with ligatures (note the `!=` and `->`):  
-![](images/sf-mono.png)  
-![](images/menlo.png)  
+Here's a couple examples of the fonts generated: SF Mono with ligatures (note the `!=` and `->`):
+![](images/sf-mono.png)
+![](images/menlo.png)
 
-**Credit**   
```

Comment:
Disable "strip trailing lines" in your editor if you save the README since this can affect the formatting in Markdown.

Reply:
Oh, I see what happened there -- I generally use blank lines as paragraph breaks, not end-of-line whitespace, since I can see the former in the editor and not the latter. I didn't realize the README used EOL whitespace instead so I didn't notice when the editor removed it.